snooty/rstspec.toml

```diff
@@ -1822,7 +1822,7 @@ realm-sdks = [
   {id = "json-expressions", title = "JSON Expressions"},
   {id = "graphql", title = "GraphQL"},
   {id = "ios", title = "Swift SDK"},
-  {id = "android", title = "Android SDK"},
+  {id = "android", title = "Java SDK"},
```

Comment:
```suggestion
  {id = "android", title = "Java SDK"},
  {id = "kotlin", title = "Kotlin SDK"},
  {id = "flutter", title = "Flutter SDK"},
```

Reply:
@i80and if it's OK, I'd also like to add two other SDKs in this PR that we haven't added in the past because they're still in alpha. We'll need them soon.

Reply:
(feel free to merge this commit in and then the PR if it looks OK to you)

Reply:
@nathan-contino-mongo Feel free to update this PR! LMK when it's ready for a re-review